### DISCUSSION TITLE
Meeting 2024-08-13

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#785] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-08-13)

We currently plan to spend some time this issue doing triage on old wg and embedded-hal issues.

### Announcements

* None yet

**Comment by burrbull** [comment]
> today's meeting

This header is little confusing.

**Reply by adamgreig** [reply]
Thanks, updated!

**Comment by adamgreig** [comment]
We discussed the proposed community survey: https://github.com/rust-embedded/wg/pull/787/

Next week we'll discuss how to promote and advertise the survey, finalise the question text, and get ready to publish it.
